_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,473 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.field-group {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .field-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+
+    &.form-group {
+
+        .field {
+            margin-bottom: spacing-vertical(x-small);
+        }
+
+        .helper-text {
+            margin-top: spacing-vertical(xx-small);
+            @include margin-left(spacing-horizontal(mid-large));
+        }
+    }
+}
+
+.form-label {
+    display: block;
+    width: auto;
+    margin-bottom: spacing-vertical(x-small);
+    font-size: font-size(base);
+    line-height: 100%;
+    color: $label-color;
+
+    // presents the label inline with the form control
+    &.label-inline {
+        display: inline-block;
+        margin-bottom: 0;
```

Comment:
Correct. I also figured that we'll address this again later when we get a grid system in place.